tests/end_to_end/serving.py

```diff
@@ -40,31 +40,23 @@ def parse_feature(data):
 client = ff.ServingClient()
 
 def serve_data():
-    for _ in range(NUMBER_OF_SLEEPS):
-        try: 
-            dataset = client.training_set(TRAININGSET_NAME, TRAININGSET_VARIANT)
-            training_dataset = dataset.repeat(10).shuffle(1000).batch(8)
-            for i, feature_batch in enumerate(training_dataset):
-                if i >= 1:
-                    return
-                print(feature_batch.to_list())
-                
-        except Exception as e:
-            print(f"\twaiting for {SLEEP_DURATION} seconds")
-            time.sleep(SLEEP_DURATION)
-    
-    raise Exception(f"Serving for {TRAININGSET_NAME}:{TRAININGSET_VARIANT} could not be completed.")
+    try:
+        dataset = client.training_set(TRAININGSET_NAME, TRAININGSET_VARIANT).wait()
+        training_dataset = dataset.repeat(10).shuffle(1000).batch(8)
+        for i, feature_batch in enumerate(training_dataset):
+            print(feature_batch.to_list())
+        
+    except Exception as e:
```

Comment:
This test wont fail if we don't raise an exception

Reply:
changed

Reply:
^